### PR TITLE
doc: add minutes for meeting June 7 2023

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2023-06-07.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+* module: make CJS load from ESM loader [#47999](https://github.com/nodejs/node/pull/47999)
+  * Added to agenda for visibility, not much to discuss
+  * Related Geoffrey, Qard added for more guidance on handling breaking changes on experimental features - <https://github.com/nodejs/TSC/issues/1397>
+
+### nodejs/TSC
+
+* CLI tools behind a Nodejs namespace [#1178](https://github.com/nodejs/TSC/issues/1178)
+  * leave on agenda until we have Daniele to confirm we have covered
+  * ok to start migrating packages to the node-core namespace.
```
Comment (MoLow) [inline meetings/2023-06-07.md:47]:
```suggestion
  * ok to start migrating packages to the node-core namespace.
  * we need to give TSC members access to npm for deprecating old packages
```

**Review by RafaelGSS — APPROVED** [review]

File: meetings/2023-06-07.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+### nodejs/admin
+
+* Archive nodejs/help in favour of a Help Section on Discussions [#802](https://github.com/nodejs/admin/issues/802)
+  * Lets defer until a future meeting where ideally we have Gireesh
+
+## Strategic Initiatives
+
+* Yagiz - Recent changes from the performance initiative:
+  * I opened a back-port for Ada v2.5 to LTS (which comes with 100% performance boost)
```
Comment (tniessen) [inline meetings/2023-06-07.md:62]: FWIW, I know that this is just quoting another comment verbatim, but we should be careful to make accurate statements about performance improvements. Are _all_ `URL` operations really twice as fast now on all platforms, especially as visible from JS? Maybe, I have no idea, but it has happened before that performance initiatives misrepresented or generalized results of microbenchmarks.

**Review by gireeshpunathil — APPROVED** [review]

File: meetings/2023-06-07.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2023-06-07.md:63]: @anonrig can you take a look and suggest updated text for this one and the later comment as well?

File: meetings/2023-06-07.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  * Khafra started writing mimesniff which will be included in the next release of Ada. <https://github.com/ada-url/mimesniff>
+  * Vinicious opened a pull request for improving the performance of legacy main resolve in module resolution: src,lib: reducing C++ calls of esm legacy main resolve node#48325
```
Comment (H4ad) [inline meetings/2023-06-07.md:65]: Vinícius, just a typo :)

Comment (mhdawson) [inline meetings/2023-06-07.md:65]:
```suggestion
* Vinícius opened a pull request for improving the performance of legacy main resolve in module resolution: src,lib: reducing C++ calls of esm legacy main resolve node#48325
```

File: meetings/2023-06-07.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+* Archive nodejs/help in favour of a Help Section on Discussions [#802](https://github.com/nodejs/admin/issues/802)
+  * Lets defer until a future meeting where ideally we have Gireesh
+
+## Strategic Initiatives
+
+* Yagiz - Recent changes from the performance initiative:
+  * I opened a back-port for Ada v2.5 to LTS (which comes with 100% performance boost)
+  * Khafra started writing mimesniff which will be included in the next release of Ada. <https://github.com/ada-url/mimesniff>
+* Vinícius opened a pull request for improving the performance of legacy main resolve in module resolution: src,lib: reducing C++ calls of esm legacy main resolve node#48325
```
Comment (anonrig) [inline meetings/2023-06-07.md:65]: The indentation of this line is wrong

File: meetings/2023-06-07.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  * Lets defer until a future meeting where ideally we have Gireesh
+
+## Strategic Initiatives
+
+* Yagiz - Recent changes from the performance initiative:
+  * I opened a back-port for Ada v2.5 to LTS (which comes with 100% performance boost)
+  * Khafra started writing mimesniff which will be included in the next release of Ada. <https://github.com/ada-url/mimesniff>
+  * Vinicious opened a pull request for improving the performance of legacy main resolve in module resolution: src,lib: reducing C++ calls of esm legacy main resolve node#48325
+  * I opened a pull request for adding NDEBUG=1 to clang flags. Feedback is appreciated. build: add -DNDEBUG=1 to clang flags node#48271
+  * Libuv 1.45.0 - 800% faster filesystem ops got merged into Node and will be included in the next release
```
[the remark this answers is not in the document]
Comment (anonrig) [reply meetings/2023-06-07.md:67]: I mostly shared those numbers for TSC members. We can remove all numeric values concerning benchmarks.

File: meetings/2023-06-07.md (hunk 2 above)
Comment (anonrig) [reply meetings/2023-06-07.md:63]: I'm currently on paternity leave but we can remove all numeric values, and use terms like "significant improvements", "major changes" etc. if not, we can just reduce the changes to "improved the performance of X"

File: meetings/2023-06-07.md (hunk 4 above)
Comment (Trott) [reply meetings/2023-06-07.md:65]:
```suggestion
  * Vinícius opened a pull request for improving the performance of legacy main resolve in module resolution: src,lib: reducing C++ calls of esm legacy main resolve node#48325
```

File: meetings/2023-06-07.md (hunk 2 above)
Comment (tniessen) [reply meetings/2023-06-07.md:63]:
```suggestion
  * I opened a back-port for Ada v2.5 to LTS (which comes with performance improvements)
```

File: meetings/2023-06-07.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (tniessen) [reply meetings/2023-06-07.md:67]:
```suggestion
  * Libuv 1.45.0 - significantly faster filesystem ops got merged into Node.js and will be included in the next release
```

**Pull request merged**